Amazon.IonDotnet/Internals/Binary/RawBinaryWriter.cs

```diff
@@ -215,7 +215,7 @@ private void PopContainer()
         private void WriteVarUint(long value)
         {
             Debug.Assert(value >= 0);
-            var written = _dataBuffer.WriteVarUint(value);
+            var written = _dataBuffer.WriteVarInt(value);
```

Comment:
Please explain this.

Reply:
I just needed `WriteVarInt` for offset. Changed it back to `WriteVarUint` here and updated the PR.
(Specifically in these 2 commits: https://github.com/amzn/ion-dotnet/pull/100/commits/02104c24215cc272a1b5586d95d69a53dfa9fcd1 and https://github.com/amzn/ion-dotnet/pull/100/commits/22274a651d145ced54572ef4de6c4ebec0ecfd3b)